src/helpers/modalSubmitActions.js

```diff
@@ -1,7 +1,17 @@
-import React from "react";
+import React, { useContext } from "react";
 import { getCurrentUser } from "./user";
 import { Auth } from "aws-amplify";
 import { MOCK_USER } from "../constants/MockUser";
+import initAlgoQuantApi from "../constants/ApiUtils";
+
+let algoquant = undefined;
```

Comment:
What is this code for ? If the api is in context I don't see why we would need to do this here.

Reply:
since this is a regular js function not a react function hooks cant be used here

Reply:
Okay could you include a comment here explaining why - just so its clear why its getting  a new API instance here rather than using the context.

Reply:
done